deploy/examples/create-external-cluster-resources.py

```diff
@@ -1727,6 +1746,7 @@ def gen_json_out(self):
                     "data": {
                         "dataPool": self.out_map["RBD_POOL_NAME"],
                         "pool": self.out_map["RBD_METADATA_EC_POOL_NAME"],
+                        "radosNamespace": self.out_map["RADOS_NAMESPACE"],
```

Comment:
What is the behavior if the rados namespace is not specified? Will this add a blank value? Seems better not to add the property if it is blank.

Reply:
can we have a design/documented steps on how this will be consumed? in any case this should not be added to the storageclass.

Reply:
How about reviewing it too at the same time https://github.com/red-hat-storage/ocs-operator/pull/2267?

Reply:
@Madhu-1 Does the csi driver support a rados namespace for consumption or that would be a new feature?

Reply:
@travisn cephcsi support rados namespace from many releases

Reply:
Remind me how the rados namespace is used, if not in the storage class?

Reply:
the user creates the RadosNamespace CR and Rook will create the RadosNamespace(when ceph cluster is created by Rook) and a unique ID is generated and its added to the CSI configmap and also to the RadosNamespace CR status , and user will create the storageclass using this clusterID and create PVC from that storageclass. https://www.mrajanna.com/Consuming-storage-from-exteranal-ceph-in-rook/ might contains all the details how to works

Reply:
Oh right, thanks for the reminder that the clusterID in the storage class just needs to be generated based on the rados namespace. Can we not generate that clusterID and put it in the storage class that the external cluster script is generating? It seems like a broken scenario if we ask for the rados namespace, but then don't generate the storage class that makes it useful. This assumes the script can generate the hash similar to [here](https://github.com/rook/rook/blob/7b57e0640fb58dbc8a2e753e62e93dc57bed7dd4/pkg/operator/ceph/pool/radosnamespace/controller.go#L337-L340).

Reply:
Yes, that is a good idea, the sad part is if we change any logic in generating clusterID that needs to be done in both places.

Reply:
>It seems like a broken scenario if we ask for the rados namespace, but then don't generate the storage class that makes it useful
We would create  storage class automatically if ask for rados-namespace, and also with the current design it adds an extra assurance that rados-namespace CR is created by reading the status. SO there is no trouble for CSI to provision